beacon-chain/state/state-native/getters_withdrawal.go

```diff
@@ -48,60 +48,62 @@ func (b *BeaconState) NextWithdrawalValidatorIndex() (primitives.ValidatorIndex,
 //
 // Spec definition:
 //
-//	def get_expected_withdrawals(state: BeaconState) -> Tuple[Sequence[Withdrawal], uint64]:
-//		epoch = get_current_epoch(state)
-//		withdrawal_index = state.next_withdrawal_index
-//		validator_index = state.next_withdrawal_validator_index
-//		withdrawals: List[Withdrawal] = []
-//		processed_partial_withdrawals_count = 0
+//		def get_expected_withdrawals(state: BeaconState) -> Tuple[Sequence[Withdrawal], uint64]:
```

Comment:
what is happening with this extra level of indentation?

Reply:
I think my IDE is applying it somehow when linting the file, will keep trying to revert